### PR TITLE
8535 discharge folio

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8535

*Description of changes:*
- add folio to discharge page with validity check

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/tests/unit/ConfirmDischarge.spec.ts [hunk 1]
```diff
@@ -91,6 +92,8 @@ describe('ConfirmDischarge registration view', () => {
     expect(wrapper.findComponent(RegistrationFee).vm.registrationType).toBe('Total Discharge')
     // buttons
     expect(wrapper.findComponent(ButtonsStacked).exists()).toBe(true)
+    // folio
+    expect(wrapper.findComponent(FolioNumberSummary).exists()).toBe(true)
```
Comment (kialj876) [inline ppr-ui/tests/unit/ConfirmDischarge.spec.ts:96]: can you add a test in here for when folio is invalid it blocks the submit?

**Review by kialj876 — COMMENTED** [review]
Hey Steph, this looks good just one comment. I gotta say though I am not a big fan of most of the formatting changes. Whatever you have running to auto fix passes our linter, but it does make it different from what we auto fix across the other apps. The main thing is when it makes unnecessary line length changes, the unneeded brackets and what it did with line 28/29/30 in ConfirmDischarge is annoying to look at for me, is it possible to adjust it for those things?

File: ppr-ui/src/components/common/FolioNumberSummary.vue [hunk 2]
```diff
@@ -1,36 +1,40 @@
 <template>
   <v-container flat class="pa-0" id="folio-summary">
```
Comment (kialj876) [inline ppr-ui/src/components/common/FolioNumberSummary.vue:2]: I also think you'll need to add the red border class somewhere in here when needed (similar to the confirm section)

File: ppr-ui/tests/unit/ConfirmDischarge.spec.ts (hunk 1 above)
Comment (jordiwes) [reply ppr-ui/tests/unit/ConfirmDischarge.spec.ts:96]: submit doesn't do anything right now.... so it's hard to tell ;)
I did put in a test for the show error property

File: ppr-ui/src/components/common/FolioNumberSummary.vue (hunk 2 above)
Comment (jordiwes) [reply ppr-ui/src/components/common/FolioNumberSummary.vue:2]: added

**Comment by jordiwes** [comment]
I fixed/reverted some of the changes that the linter made

File: ppr-ui/tests/unit/ConfirmDischarge.spec.ts (hunk 1 above)
Comment (kialj876) [reply ppr-ui/tests/unit/ConfirmDischarge.spec.ts:96]: ah but it does

**Review by kialj876 — APPROVED** [review]
thanks for updating!

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**